tools/distributions-scraper/scraper/debian.py

```diff
@@ -0,0 +1,88 @@
+import base64
+import requests
+from email.parser import Parser
+from scraper.base import BaseScraper
+
+
+def _fetch_items(codename, version):
```

Comment:
This function is pretty long, and has 5 levels of nesting. I know this is only adjacent to Multipass, but it will need to be read and maintained. Would you mind splitting the code and organizing into smaller, adequately named units, please? It should then be easier to quickly understand the different parts of the procedure and how they relate to each other.